src/app/content/hooks/locationChange/index.ts

```diff
@@ -1,14 +1,29 @@
+import { user as userSelector } from '../../../auth/selectors';
 import { RouteHookBody } from '../../../navigation/types';
+import { closeCallToActionPopup, openCallToActionPopup } from '../../actions';
 import { loadHighlights } from '../../highlights/hooks';
 import { content } from '../../routes';
 import { syncSearch } from '../../search/hooks';
+import { showCTAPopup } from '../../selectors';
 import resolveContent from './resolveContent';
 
 const hookBody: RouteHookBody<typeof content> = (services) => async(action) => {
   await resolveContent(services, action.match);
   const search = syncSearch(services)(action);
   const highlights = loadHighlights(services)();
 
+  const state = services.getState();
```

Comment:
this logic is ok, but can we make a separate hook to manage it?

Reply:
Will this require to wait for `await resolveContent(services, action.match);` in both of those hooks?

Reply:
no, the logic you're adding doesn't care if the book is loaded or not, so it shouldn't matter. you will have to add something to make sure the user query is finished, i think we have something for that somewhere

Reply:
looks like in order to accomplish this you might want to actually hook into `receiveLoggedOut`

Reply:
The specs say:
> banner appears 5s after page load with some kind of animation

This is why I've dispatched this after receiveContent

Would you like me still to hook this to `receiveLoggedOut` - I assume this hook should be placed in `/content/hooks/` rather than `/auth/hooks/`?

Reply:
Yea when they say page load they mean after the initial page load not after everything is loaded. 

Yes you should hook into receiveloggedout and put that in content, because it is controlling content stuff. You'll need to so something else with your timer too, because it should be 5 seconds from page load not 5 seconds from the user query finishing